media/css/new-start-style.css

```diff
@@ -215,11 +216,12 @@ html[data-channel="daily"] .logo-img {
   justify-self: end;
   background-color: #ff0039;
   background-image: linear-gradient(#FF555F, #D70022);
-  border-radius: .25rem;
+  border-radius: .12rem;
```

Comment:
Nit: border radius should be a fixed value since we don't want to change based on the font size.
Let's make it 3px.